proto/buf/alpha/registry/v1alpha1/resolve.proto

```diff
@@ -114,13 +124,54 @@ message GetLocalModulePinsResponse {
   repeated buf.alpha.module.v1alpha1.ModulePin dependencies = 2;
 }
 
-message GetRemotePackageVersionRequest {
+message GetGoVersionRequest {
+  // The plugin reference to resolve.
+  GetRemotePackageVersionPlugin plugin_reference = 1;
+  // The module reference to resolve.
+  LocalModuleReference module_reference = 2;
+}
+
+message GetGoVersionResponse {
+  // version is the resolved version to be used with the go module proxy.
+  string version = 1;
+}
+
+message GetMavenVersionRequest {
+  // The plugin reference to resolve.
+  GetRemotePackageVersionPlugin plugin_reference = 1;
+  // The module reference to resolve.
+  LocalModuleReference module_reference = 2;
+}
+
+message GetMavenVersionResponse {
+  // version is the resolved version to be used with the maven registry.
```

Comment:
```suggestion
  // version is the resolved version to be used with the maven repository.
```

Reply:
We use the term `registry` everywhere don't we?

Reply:
nvm apparently it's different for maven